src/openrct2/CmdlineSprite.cpp

```diff
@@ -621,7 +621,7 @@ int32_t cmdline_for_sprite(const char** argv, int32_t argc)
             // Get palette option, if present
             bool keep_palette = false;
             json_t* palette = json_object_get(sprite_description, "palette");
-            if (palette && json_is_string(palette))
+            if (json_is_string(palette))
```

Comment:
There are a few more like this

Reply:
On the same file, I actually found no other `A && json_is_\(.*\)(A)` combination

Reply:
Yeah don't use regex for such things instead read the actual file maybe. See line 610

Reply:
To be fair, line 610 is an `||` clause, rather than an `&&` :P